src/BitmovinEncodingLambda/index.js

```diff
@@ -1,3 +1,32 @@
+import mysql from 'mysql';
+
+function storeBitmovin(input, folder) {
+  const livestream_dir = 'bitmovin/' + folder
+  console.log('livestream_dir', livestream_dir)
+  console.log(`input: ${input}`);
+  updateAsset(input, livestream_dir);
+}
+
+function updateAsset(input, livestream_dir) {
+  const connection = mysql.createConnection({
+    host: process.env.DB_HOST,
```

Comment:
@ilyakatz recreated PR without .webpack handler.js.  I'm not going to add dotenv to package.json because im only using it locally.  I will add these variables to the code configuration in the lambda web console.  I can add notes to the readme to describe where they are.